core/server/master/src/main/java/alluxio/master/block/WorkerRegisterContext.java

```diff
@@ -81,7 +85,7 @@ public boolean isOpen() {
   }
 
   void updateTs() {
-    mLastActivityTimeMs = Instant.now().toEpochMilli();
+    mLastActivityTimeMs = mClock.millis();
```

Comment:
Don't `mClock.millis()` and `Instant.now().toEpochMillis()` return the same value?

Reply:
The difference is, now the WorkerRegisterContext takes the clock from outside so there's a way we control the clock. The externally-passed clock is from the BlockMaster, so now there's a way to manually control the BlockMaster and WorkerRegisterContext clock at the same time. In integration tests like BlockMasterRegisterStreamIntegrationTest(and some others), we typically create a `ManualClock` and control the time instead of relying on system time and `sleep()`. 

In the flaky `BlockMasterRegisterStreamIntegrationTest`, the BlockMaster is using the manual clock and the `WorkerRegisterContext` using `Instant.now()`. So when these two clocks don't align well, the behavior is undefined.

In a real cluster, both clocks will be the JVM SystemClock. So in a real cluster `mClock` will be equivalent to `Instant.now()`. This fix does not introduce any behavior change in a real cluster.